docs/ops/image/Interpolate_4.md

```diff
@@ -15,7 +15,7 @@
   * **Type**: string
   * **Default value**: none
   * **Required**: *yes*
-  * **Note**: Only 2D and 4D tensors with `axes = {0, 1}` and `axes = {2, 3}` respectively are supported for `"mode" == "linear_onnx"`.
+  **Note**: Only 2D, 3D, 4D, 5D tensors with `axes = {0, 1}`, `axes = {0, 1, 2}`, `axes = {2, 3}`,  `axes = {2, 3, 4}` respectively are supported for `"mode" == "linear_onnx"`.
```

Comment:
@lazarevevgeny  Please take a look.

Reply:
This mode corresponds to the linear mode in the ONNX sense, and ONNX supports only input ranks 2, 3, 4, 5. Next, 

1. ONNX interprets 2D tensors of the shape `[H, W]` as 4D tensors with the shape `[1, 1, H, W]`, in the ONNX Resize with linear mode. Hence, correct axes are only `[0, 1]`, for 2D tensors.
2. ONNX interprets 3D tensors of the shape `[D, H, W]` as 5D tensors with the shape `[1, 1, D, H, W]`, in the ONNX Resize with linear mode. Hence, correct axes are only `[0, 1, 2]`, for 3D tensors.
3. For 4D tensors, ONNX supports only interpolation axes `[2, 3]`.
4. For 5D tensors, ONNX supports only interpolation axes `[2, 3, 4]`.

Consequently, the sentence
> Only 2D, 3D, 4D, 5D tensors with `axes = {0, 1}`, `axes = {0, 1, 2}`, `axes = {2, 3}`,  `axes = {2, 3, 4}` respectively are supported for `"mode" == "linear_onnx"`.

is  correct.